### PR TITLE
Rename master to main in all instructions and config

### PR DESCRIPTION
**Pull request by brianamarie**
This pull request changes references to the `master` branch to the `main` branch.

Once this is ready to merge, I will:

- [x] Rename the default branch in this repository
- [x] Update any other PRs in this repository
- [x] Rename the default branch in the [template repository](https://github.com/githubtraining/caption-this-template)
- [ ] Merge this PR
- [ ] Test to make sure things work as expected

🎉 

cc @githubtraining/programs - the renaming process took me about 10 minutes for the files, and once it's reviewed, I'll keep an eye on how long the other tasks take as well for planning the other courses.

### CODE REVIEW

**Comment by suresh672** [comment]
Facebook phishing page kase banau

**Comment by jedpepper** [comment]
Good work guys

File: responses/05_open-a-pr.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ Let’s think back to the GitHub flow again. You have created a branch, added a
 <hr>
 </details>
```
Comment (Shaker5191) [inline responses/05_open-a-pr.md:15]: - #_****
```suggestion
 </details>

```
****_

**Review by Shaker5191 — COMMENTED** [review]
fe173a04d487a85199a0996eafb5729cea69db16

**Pull request merged**